database/G/George Thorogood And The Destroyers/Maverick/I Drink Alone

```diff
@@ -0,0 +1,54 @@
+I drink alone
+Ye-yeah, with nobody else
+I drink alone
+Ye-yeah, with nobody else
+Yeah, you know when I drink alone
+I prefer to be by myself
+
+Now every morning just before breakfast
+I don’t want no coffee or tea
+Just me and good buddy Wiser
+That’s all I ever need
+’Cause I drink alone, yeah
+With nobody else
+Yeah, you know when I drink alone
+I prefer to be by myself
+
+Yeah, the other night I laid sleeping
+And I woke from a terrible dream
+So I caught up my pal Jack Daniel
+And his partner Jimmy Beam
+And we drank alone
+Ye-yeah, with nobody else
+Yeah, you know when I drink alone
+I prefer to be by myself
+
+Yeah, the other day I got invited to a party
+But I stayed home instead
+Just me and my pal Johnnie Walker
+And his brothers Black and Red
+And we drank alone
+Ye-yeah, with nobody else
+Yeah, you know when I drink alone
+I prefer to be by myself
+
+Yeah, my whole family done give up on me
+And it makes me feel, oh, so bad
+The only one who will hang out with me
+Is my dear old granddad
```

Comment:
Is "granddad" supposed to be some alcohol reference/pun, as everywhere above?

Reply:
Looked it up, there's <https://en.wikipedia.org/wiki/Old_Grand-Dad>. So perhaps should be "Is my dear Old Grand-Dad".

Reply:
I thought it was referring to his actual grandfather, since on :35 he mentions his family being "done" with him, but that's nice that it's a pun of sorts... good find!